QuantLib/ql/models/model.cpp

```diff
@@ -35,23 +38,24 @@ namespace QuantLib {
     CalibratedModel::CalibratedModel(Size nArguments)
     : arguments_(nArguments),
       constraint_(new PrivateConstraint(arguments_)),
-      shortRateEndCriteria_(EndCriteria::None) {}
+      endCriteria_(EndCriteria::None) {}
```

Comment:
Same here. It should have been called `endCriteria_`, but someone might have inherited from `CalibrationHelper` and used the current name.